### PR TITLE
lightningd: implement --log-timestamps=false.

### PR DESCRIPTION
**Pull request by rustyrussell**
Fixes: #4494
Signed-off-by: Rusty Russell <rusty@rustcorp.com.au>
Changelog-Added: config: New option `log-timestamps` allow disabling of timestamp prefix in logs.

### CODE REVIEW

File: doc/lightningd-config.5 [hunk 1]
```diff
@@ -197,6 +197,11 @@ Log to this file instead of stdout\. Sending \fBlightningd\fR(8) SIGHUP will
 cause it to reopen this file (useful for log rotation)\.
 
 
+ \fBlog-timetamps\fR=\fIBOOL\fR
+Set this to false to turn of timestamp prefixes (they will still appear
```
Comment (jsarenik) [inline doc/lightningd-config.5:201]: to turn off

File: doc/lightningd-config.5.md [hunk 2]
```diff
@@ -157,6 +157,10 @@ with multiple daemons.
 Log to this file instead of stdout. Sending lightningd(8) SIGHUP will
 cause it to reopen this file (useful for log rotation).
 
+ **log-timetamps**=*BOOL*
+Set this to false to turn of timestamp prefixes (they will still appear
```
Comment (jsarenik) [inline doc/lightningd-config.5.md:161]: "to turn off", ah this is the source and the other is generated from this

**Review by jsarenik — CHANGES_REQUESTED** [review]
Thank you Rusty!

File: doc/lightningd-config.5 (hunk 1 above)
Comment (jsarenik) [reply doc/lightningd-config.5:201]: see below (this one is generated)

**Comment by cdecker** [comment]
Just a minor typo in the docs.

ACK https://github.com/ElementsProject/lightning/pull/4504/commits/7e8d771acfd595260f802f1fd973f09421570c62

**Review by cdecker — APPROVED** [review]

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4504/commits/7e8d771acfd595260f802f1fd973f09421570c62

**Comment by rustyrussell** [comment]
Typo fixed, thanks!

**Comment by jsarenik** [comment]
ACK 451e1898c

**Review by jsarenik — APPROVED** [review]

**Pull request merged**